content/sensu-go/5.15/guides/use-federation.md

```diff
@@ -0,0 +1,313 @@
+- [Step 5 Register clusters](#step-5-register-clusters)
+  - [Register a single cluster](#register-a-single-cluster)
+  - [Register additional clusters](#register-additional-clusters)
+- [Step 6 Get a unified view of all your clusters in the web UI](#step-6-get-a-unified-view-of-all-your-clusters-in-the-web-ui)
+- [Next steps](#next-steps)
+
+**COMMERCIAL FEATURE**: Access federation in the packaged Sensu Go distribution. For more information, see the [getting started guide][8].
+
+Sensu's [federation API][1] allows you to register external clusters, access resources across multiple clusters via the web UI, and mirror your changes in one cluster to follower clusters.
+
+Federation is not enabled by default. You must create a cluster resource for the federation cluster and [register it](#register-a-single-cluster).
+
+Create, update, and delete clusters using sensuctl [create][5], [edit][6], and [delete][7] commands. Only cluster administrators can register a new cluster, but every user can [query the list of clusters][11].
+
+## What you can do with federation
+
+Federation affords visibility into the health of your infrastructure and services across multiple distinct Sensu instances within a single web UI. This is useful when you want to provide a single entry point for Sensu users who need to manage monitoring across multiple distinct physical data centers, cloud regions or providers.
+
+## Configuring federation
+
+Complete federation of multiple Sensu instances is accomplished through a combination of features:
+
+| Feature                             | Provides |
+|-------------------------------------|--------------------------------------------------------------------|
+| JSON Web Token authentication       | Cross-cluster token authentication using asymmetric key encryption |
+| Etcd Replicators                    | Replication of RBAC policy across clusters and/or namespaces       |
+| Federation Gateway and APIs         | Configure federation access for cross-cluster visibility in web UI |
+
+The following steps are required to configure these features for unified visibility in a single web UI. For our examples, we'll assume there are three named Sensu clusters we wish to federate:
+
+* `gateway`
+* `alpha`
+* `beta`
+
+The `gateway` cluster will be the entry-point for operators to manage Sensu resources in the `alpha` and `beta` clusters.
+
+### Step 1 Configure backends for TLS
+
+As federation depends on communication between multiple disparate clusters, working TLS is a prerequisite for successful configuration. This guide assumes that you have provided each backend member with TLS credentials (key and certificate), as well as the CA certificate chain required for one Sensu backend to validate the certificates presented by other backends. If you don't have existing infrastructure for issuing certificates, see our Securing Sensu guide for [our recommended self-signed certificate issuance process][13].
```

Comment:
```suggestion
Because federation depends on communication with multiple disparate clusters, working TLS is required for successful configuration. This guide assumes that you have provided each backend member with TLS credentials (key and certificate), as well as the CA certificate chain required for one Sensu backend to validate the certificates presented by other backends. If you don't have existing infrastructure for issuing certificates, see [Securing Sensu][13] for our recommended self-signed certificate issuance process.
```